### PR TITLE
Don't auto-update jackson-datatype

### PR DESCRIPTION
**Pull request by raboof**
Instead of #932

### CODE REVIEW

File: .scala-steward.conf [hunk 1]
```diff
@@ -8,6 +8,7 @@ updates.ignore = [
 updates.pin = [
   # To be updated in tandem with upstream Akka
   {groupId = "com.fasterxml.jackson.core", version = "2.10."}
+  {groupId = "com.fasterxml.jackson.datatype", version = "2.10."}
```
Comment (patriknw) [inline .scala-steward.conf:11]: Isn't `master` for Akka 2.6.x only? Then it should align the Jackson version (in Dependencies.scala) with what we have in Akka, which is 2.11.4

Also, see https://github.com/akka/akka/blob/master/.scala-steward.conf for pin and ignore

File: project/Dependencies.scala [hunk 2]
```diff
@@ -18,19 +18,18 @@ object Dependencies {
...
-    "com.fasterxml.jackson.core" % "jackson-databind" % JacksonDatabindVersion
+    "com.fasterxml.jackson.core" % "jackson-databind" % JacksonVersion
   )
 
   private val JacksonDatatype = Seq(
     "com.fasterxml.jackson.datatype" % "jackson-datatype-guava" % JacksonVersion,
     "com.fasterxml.jackson.datatype" % "jackson-datatype-jdk8" % JacksonVersion,
     // Specifying guava dependency because older transitive dependency has security vulnerability
-    "com.google.guava" % "guava" % "27.1-jre"
+    "com.google.guava" % "guava" % "29.0-jre"
```
Comment (patriknw) [inline project/Dependencies.scala:32]: Could we remove this now and rely on the transitive dependency instead?

Comment (raboof) [reply project/Dependencies.scala:32]: I don't think so: https://repo1.maven.org/maven2/com/fasterxml/jackson/datatype/jackson-datatype-guava/2.11.4/jackson-datatype-guava-2.11.4.pom still specifies 0.20

Comment (patriknw) [reply project/Dependencies.scala:32]: ok, hopefully switching from 27 to 29 doesn't break something else

**Review by patriknw — APPROVED** [review]
LGTM

File: project/Dependencies.scala (hunk 2 above)
Comment (raboof) [reply project/Dependencies.scala:32]: `com.orbitz.consul:consul-client:1.5.3` was already bringing in version 29.0-jre, so sbt users would already get that one. I updated this one here so maven users are in sync again.

**Pull request merged**